platform-hub-api/app/services/feature_flag_service.rb

```diff
@@ -0,0 +1,43 @@
+module FeatureFlagService
+  extend self
+
+  # Feature flags data is stored in HashRecord as:
+  # data: {
+  #   'your-feature-flag-name': Bool,
+  #   ...
+  # }
+
+  def all
+    feature_flags.data
+  end
+
+  def create_or_update(flag, value)
+    f = feature_flags
+    f.with_lock do
+      f.data[flag.to_s] = value
+      f.save!
+      "Created/updated feature flags"
+    end
+  end
+
+  def delete(flag)
+    f = feature_flags
+    f.with_lock do
+      f.data.reject! {|k,_| k == flag.to_s}
+      f.save!
+      "Deleted feature flag"
+    end
+  end
+
+  def is_enabled?(flag)
+    feature_flags.data.with_indifferent_access[flag] || false
```

Comment:
Unless the key doesn't exist in the first place (which will be the case when first used (since we said the frontend would control the actual flag data).